lib/ApplicationFeatures/EnvironmentFeature.cpp

```diff
@@ -153,6 +153,15 @@ void EnvironmentFeature::prepare() {
     // file not found or value not convertible into integer
   }
 
+  // Report memory found:
+  uint64_t ram = PhysicalMemory::getValue();
+  std::string overriddenmsg;
+  if (PhysicalMemory::overridden()) {
+    overriddenmsg = " (overridden by environment variable)";
+  }
+  LOG_TOPIC("25362", INFO, Logger::MEMORY)
+  << "Available physical memory: " << ram << overriddenmsg;
+
```

Comment:
The `PhysicalMemory` namespace does not exist in 3.6. It was introduced in 3.7, so this will not work.

Reply:
Sorry, created PR before compilation finished. See latest push.